scan/src/images/Images.re

```diff
@@ -66,6 +66,8 @@
 [@bs.module] external ledgerStep2Cosmos: string = "./ledger-step2-cosmos.svg";
 [@bs.module] external ledgerStep2BandChain: string = "./ledger-step2-bandchain.svg";
 [@bs.module] external sort: string = "./sort.svg";
+[@bs.module] external mobileSort: string = "./mobile-sort-icon.svg";
+[@bs.module] external mobileSortActive: string = "./mobile-sort-icon-active.svg";
```

Comment:
Do they mean Asc and Desc? If they do, let's use `mobileSortAsc` and `mobileSortDesc` name.

Reply:
Nope, mobileSort for normal state (grey color) and mobileSortActive for active state(blue color).